catalog/ui/catalog-ui-search/src/test/java/org/codice/ddf/catalog/ui/forms/SearchFormsLoaderTest.java

```diff
@@ -13,55 +13,80 @@
  */
 package org.codice.ddf.catalog.ui.forms;
 
-import static java.lang.String.format;
-import static junit.framework.TestCase.fail;
 import static org.hamcrest.MatcherAssert.assertThat;
 import static org.hamcrest.collection.IsCollectionWithSize.hasSize;
 import static org.hamcrest.core.Is.is;
 
+import ddf.catalog.CatalogFramework;
 import ddf.catalog.data.Metacard;
-import java.io.File;
 import java.net.URL;
-import java.nio.file.Paths;
 import java.util.List;
 import org.codice.ddf.catalog.ui.forms.data.AttributeGroupMetacard;
 import org.codice.ddf.catalog.ui.forms.data.QueryTemplateMetacard;
+import org.codice.ddf.catalog.ui.util.EndpointUtil;
 import org.junit.Test;
 import org.junit.runner.RunWith;
 import org.junit.runners.JUnit4;
+import org.mockito.Mock;
 
 @RunWith(JUnit4.class)
 public class SearchFormsLoaderTest {
+  @Mock CatalogFramework catalogFramework;
```

Comment:
The `@Mock` annotation won't do anything without a `@RunWith(MockitoJUnitRunner.class)` or a call to `initMocks(this)`.